### ISSUE TITLE
[Enhancement]: Redirect users to forum on account confirmation emails instead of Discord for support 

### ISSUE THREAD

**Issue by ExperiencersInternational** [PretendoNetwork/account#127]
### Checked Existing

- [X] I have checked the repository for duplicate issues.

### What enhancement would you like to see?

When confirming a new account, users are currently redirected to the Discord server if they need assistance. But since we're looking to move support over to the forum, we should redirect there instead.

![Screenshot_20241127-145416](https://github.com/user-attachments/assets/d58a09e1-0483-4e77-a462-eeb9b15638d6)

![Screenshot_20241127-150039](https://github.com/user-attachments/assets/96666c44-d1f1-4cbe-a632-49bbf54d26a9)

**Comment by binaryoverload** [comment]
We are currently in different stages of providing support, so I don't think we should remove Discord here. But, I agree adding the forum would be a good addition, so I think we can rephrase this to be something like

"please contact us on the **Forum** or by joining our **Discord Server**"

**Comment by AToska21** [comment]
> We are currently in different stages of providing support, so I don't think we should remove Discord here. But, I agree adding the forum would be a good addition, so I think we can rephrase this to be something like
> 
> "please contact us on the **Forum** or by joining our **Discord Server**"

I'll make a PR for it!
